swap-both-endpoints/action.yaml

```diff
@@ -0,0 +1,19 @@
+name: 
```

Comment:
issue: Can we re-name this file to include dev as it uses the dev runner?

Reply:
Decided to directly invoke the endpoint swap workflow instead of create a workflow pegged to a specific runner / env combo